generators/server/templates/pom.xml.ejs

```diff
@@ -125,6 +125,11 @@
         <mapstruct.version>1.3.0.Final</mapstruct.version>
 <%_ if (enableSwaggerCodegen) { _%>
         <jackson-databind-nullable.version>0.2.0</jackson-databind-nullable.version>
+<%_ } _%>
+<%_ if (cacheProvider === 'caffeine') { _%>
+        <caffeine.version>2.8.0</caffeine.version>
```

Comment:
Does this property is really used?

Reply:
Good catch. I've corrected it by defining caffeine.version and jcache.version explicitly in the dependencies section. Please take a look and let me know if you have any further issues.